docker/docker_entrypoint.sh

```diff
@@ -6,4 +6,6 @@
 python manage.py migrate                  # Apply database migrations
 python manage.py collectstatic --noinput  # Collect static files
 
+bash api_app/script_analyzers/yara_repo_downloader.sh
```

Comment:
should revert this change. The yara/quark-engine rules are to be kept constant for a particular IntelOwl release (otherwise, there are bugs). If we make this script part of entrypoint, then it gets executed each time the container is started thus slowing it down by a lot.

Reply:
that's right... the idea was to reduce the size of the base image + reduce the build time of the image.

I would like to avoid to load the rules in any case at the start. I think they should be loaded only the first time they are requested and only if the users want to use them. Another motivation: the number of Yara repositories is growing a lot so I do not think they should be managed in the base image. It does not scale. To solve this we would need an additional work that is maybe out of the scope of the PR.

We could just create an issue for this and revert this change. Thoughts?

Reply:
> For now, we could just create an issue for this and revert this change.

I agree.

> To solve this we would need an additional work that is maybe out of the scope of the PR.

Yeah